src/ms.tex

```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
 % Transition of needs: From new features, to community development, sustainability, ...
 % - pipeline between User, Participant, Code contributor, Maintainer, Coordinator.
 
+\subsection{Inclusion, Diversity, and Equity Programs} \label{sec:project-ide}
+
+\secauthor{Lía Corrales}
 
 
 \subsection{Funding} \label{sec:project-funding}
 
-New funding sources.
-% Funding received:
-% - Moore
-% - NASA
+\secauthor{Aarya Patil}
+
+Summarize funding sources (Moore, NASA) and amounts and what this has been used
+for.
 
 
 \section{Supporting the Ecosystem of Astronomical Python Software}
 \label{sec:ecosystem}
 
 \subsection{Community-oriented infrastructure}
 
-Infrastructure packages that exist to support and help others maintain their package infrastructure.
+\secunfilled
+
+Infrastructure packages that exist to support and help others maintain their
+package infrastructure.
 
 \subsection{Affiliated packages}
 
-New packages and major updates!
+\secauthor{Matt Craig, Brett Morris}
+
+Highlight a few new affiliated packages and major updates to existing ones.
+Include a big table of all affiliated packages and references (as in v2.0
+paper).
+
+\subsection{Connections with data archives}
 
-\subsection{Connections with observatories}
+\secauthor{Adam Ginsburg}
```

Comment:
@keflavich : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.